addons/binding/pom.xml

```diff
@@ -84,6 +84,7 @@
     <module>org.openhab.binding.samsungtv</module>
     <module>org.openhab.binding.sensebox</module>
     <module>org.openhab.binding.silvercrestwifisocket</module>
+    <module>org.openhab.binding.sinope</module>
```

Comment:
Please also add the binding to https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml